### PR TITLE
Docs: Add Volta to Grafana dev setup instructions

### PR DESCRIPTION
**Pull request by Elfo404**
Following up on a recent discussion this PR updates the developer guide adding links to volta setup instruction.

Additionally, it fixes volta's config for all workspaces
- input datasource
- admin app

And adds it to all packages in `packages/*`.

It also changes node version to match exactly the one in `Dockerfile`.

**NOTE:**
we have some inconsistencies in our docker files.
- Our main one (`Dockerfile`) uses node `14.16.0`
- `Dockerfile.ubuntu` uses `14.15.1`
- `scripts/build/ci-e2e/Dockerfile` uses `12.19.0`

We might want to go through and review them.

Also, in the developer guide is mentioned this awesome blog post by @thisisobate: https://grafana.com/blog/2021/03/03/how-to-set-up-a-grafana-development-environment-on-a-windows-pc-using-wsl/, maybe we can update it as well?

### CODE REVIEW

File: contribute/developer-guide.md [hunk 1]
```diff
@@ -10,8 +10,14 @@ Make sure you have the following dependencies installed before setting up your d
 
 - [Git](https://git-scm.com/)
 - [Go](https://golang.org/dl/) (see [go.mod](../go.mod#L3) for minimum required version)
-- [Node.js (Long Term Support)](https://nodejs.org)
-- [Yarn](https://yarnpkg.com)
+
+Additionally you'll need:
+
+- [Node.js](https://nodejs.org) (see `volta.node` property in [package.json](../package.json) for the correct version)
```
Comment (achatterjee-grafana) [inline contribute/developer-guide.md:16]:
```suggestion
- [Node.js](https://nodejs.org) (see `volta.node` property in [package.json](../package.json) for the correct version).
```

File: contribute/developer-guide.md [hunk 2]
```diff
@@ -10,8 +10,14 @@ Make sure you have the following dependencies installed before setting up your d
...
+- [Yarn](https://yarnpkg.com) (see `volta.yarn` property in [package.json](../package.json) for the correct version)
```
Comment (achatterjee-grafana) [inline contribute/developer-guide.md:17]:
```suggestion
- [Yarn](https://yarnpkg.com) (see `volta.yarn` property in [package.json](../package.json) for the correct version).
```

File: contribute/developer-guide.md [hunk 3]
```diff
@@ -10,8 +10,14 @@ Make sure you have the following dependencies installed before setting up your d
...
+
+We recommend using [Volta](https://github.com/volta-cli/volta) to manage your JS toolchain.
+Refer to the [Volta Getting Started Guide](https://docs.volta.sh/guide/getting-started) for setup instructions for you operating system
```
Comment (achatterjee-grafana) [inline contribute/developer-guide.md:20]:
```suggestion
Refer to the [Volta Getting Started Guide](https://docs.volta.sh/guide/getting-started) for setup instructions for you operating system.
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Added doc change request.

File: contribute/developer-guide.md [hunk 4]
```diff
@@ -10,8 +10,14 @@ Make sure you have the following dependencies installed before setting up your d
...
-- [Node.js (Long Term Support)](https://nodejs.org)
-- [Yarn](https://yarnpkg.com)
+
+Additionally you'll need:
+
+- [Node.js](https://nodejs.org) (see `volta.node` property in [package.json](../package.json) for the correct version).
+- [Yarn](https://yarnpkg.com) (see `volta.yarn` property in [package.json](../package.json) for the correct version).
+
+We recommend using [Volta](https://github.com/volta-cli/volta) to manage your JS toolchain.
+Refer to the [Volta Getting Started Guide](https://docs.volta.sh/guide/getting-started) for setup instructions for you operating system.
```
Comment (oddlittlebird) [inline contribute/developer-guide.md:20]:
```suggestion
Refer to the [Volta Getting Started Guide](https://docs.volta.sh/guide/getting-started) for setup instructions for your operating system.
```

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Review by aknuds1 — APPROVED** [review]

**Review by thisisobate — APPROVED** [review]
LGTM! 🎉

**Pull request merged**